### ISSUE TITLE
User can log in but not recorded as OpenTree participant

### ISSUE THREAD

**Issue by kcranston** [OpenTreeOfLife/opentree#882]
See https://tree.opentreeoflife.org/curator/profile/wdahdul

Although @wdahdul can log in and edit studies. Example of [recent phylesystem commit](https://github.com/OpenTreeOfLife/phylesystem-1/commit/1cdf85e6ff1da227d3eb1fe8ee3d6a43e4ef694f).

Any idea what is going on here? This prevents editing collections.

**Comment by snacktavish** [comment]
I think is is similar to something we ran into with Romina - if a user doesn't have a public name, phylesystem commits are not actually associated with their github account (see link on https://github.com/OpenTreeOfLife/phylesystem-1/commit/80e8bf4c5b2f336d128a55fa2474a60be6f70c8f vs https://github.com/OpenTreeOfLife/phylesystem-1/commit/89ba9d059a1847abb68efa74d55a85d0e525e443)

**Comment by kcranston** [comment]
This likely related to #760 although the difference here is that we get the right username but the email appears to be invalid (see the info icon on the phylesystem commit).

**Comment by kcranston** [comment]
Wondering if our documentation / validation / error messages need to be more specific about entering information in your github profile.

**Comment by kcranston** [comment]
Ok, so the issue is that GitHub cannot associate activity with a user unless the user has a valid email address in their GitHub profile. Note that the email address does **not** need to be public.

**Comment by kcranston** [comment]
Seems to be more complicated than this - we have many users without public addresses who have profile pages. Perhaps related to user having the "Keep email private" setting in GitHub profile?

**Comment by kcranston** [comment]
I think we have finally figured this out. There are two kinds of OpenTree activity that trigger generation of a profile page: 
* Feedback left on the site. We save feedback as [issues in the feedback repo](https://github.com/OpenTreeOfLife/feedback/issues) through the GitHub API. In this case, activity is associated with a user's GitHub account no matter what email settings they have. 
* Curation activity. In this case, we use our own [phylesystem-api](https://github.com/OpenTreeOfLife/phylesystem-api). The behaviour here is similar to local git - activity needs an email address, but in this case the email needs to be public. See this [GitHub help article](https://help.github.com/articles/why-are-my-contributions-not-showing-up-on-my-profile/) for details. 

I've updated the wiki page about user accounts:
https://github.com/OpenTreeOfLife/opentree/wiki/OpenTree-user-accounts-and-GitHub
